framework/src/outputs/Console.C

```diff
@@ -366,7 +366,7 @@ Console::output(const ExecFlagType & type)
       _old_nonlinear_norm = std::numeric_limits<Real>::max();
 
     _console << std::setw(2) << _nonlinear_iter
-             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << '\n';
+             << " Nonlinear |R| = " << outputNorm(_old_nonlinear_norm, _norm) << std::endl;
```

Comment:
> The flush in action is more important to me.

I do not have a problem with that one.

> Also we are not flushing files but just screen. The impact should be very small.

1) On clusters where you are running a batch job, the screen output is redirected to a file. 2) Terminal emulators are not that fast as you would think. 3) Console can be easily redirected to a file via 2 parameters, so your assumption that we are flushing just screen is not correct.